### PR TITLE
Fixed visitor to extract expression from paren statements.

### PR DESCRIPTION
**Pull request by DeathByTape**
Addresses: https://github.com/yahoo/elide/issues/210

### CODE REVIEW

**Comment by DeathByTape** [comment]
Addresses: https://github.com/yahoo/elide/issues/210

**Comment by yahoocla** [comment]
CLA is valid!

File: elide-core/src/main/antlr4/com/yahoo/elide/generated/parsers/Expression.g4 [hunk 1]
```diff
@@ -24,7 +24,7 @@ Doesn't work with single character names.
 
 expression
     : NOT WS expression                             #NOT
-    | LPAREN WS? expression WS? RPAREN              #PAREN
+    | LPAREN WS* expression WS* RPAREN              #PAREN
```
Comment (clayreimann) [inline elide-core/src/main/antlr4/com/yahoo/elide/generated/parsers/Expression.g4]: Let's make it `WS*` all the whitespace

File: elide-core/src/main/antlr4/com/yahoo/elide/generated/parsers/Expression.g4 [hunk 2]
```diff
@@ -23,11 +23,11 @@ Doesn't work with single character names.
...
 expression
-    : NOT WS expression                             #NOT
-    | LPAREN WS? expression WS? RPAREN              #PAREN
-    | left=expression WS AND WS right=expression    #AND
-    | left=expression WS  OR WS right=expression    #OR
-    | permissionClass                               #PERMISSION
+    : NOT WS expression                               #NOT
+    | LPAREN WS* expression WS* RPAREN                #PAREN
+    | left=expression WS* AND WS* right=expression    #AND
+    | left=expression WS*  OR WS* right=expression    #OR
```
Comment (clayreimann) [inline elide-core/src/main/antlr4/com/yahoo/elide/generated/parsers/Expression.g4]: `NOT`, `AND`, and `OR` need to be `WS+`

**Comment by clayreimann** [comment]
👍

**Pull request merged**